.dotnet/src/Custom/Chat/FunctionChatMessage.cs

```diff
@@ -20,12 +20,8 @@ public partial class FunctionChatMessage : ChatMessage
     /// <param name="functionName">
     ///     The name of the called function that this message provides information from.
     /// </param>
-    /// <param name="content">
-    ///     The textual content that represents the output or result from the called function. There is no format
-    ///     restriction (e.g. JSON) imposed on this content.
-    /// </param>
-    public FunctionChatMessage(string functionName, string content = null)
-        : base(ChatMessageRole.Function, content)
+    public FunctionChatMessage(string functionName)
```

Comment:
I just noticed that `content` is actually a required parameter here (although it is nullable and that's probably the reason why it is confusing). Could you restore the `content` parameter but removed its default value of `null`? i.e.:

```csharp
public FunctionChatMessage(string functionName, string content)
```

And then, you can also restore the `CreateFunctionMessage` factory method in the Custom/Chat/ChatMessage.cs file. That one was correct in that it was not defaulting the `content` parameter to `null`.

Reply:
Ah..the default value was null here, so thought it's an optional parameter.